src/main/resources/schemas/nanopore-instrument-output.schema.json

```diff
@@ -407,6 +438,9 @@
                   },
                   {
                     "$ref": "#/definitions/throughput_log"
+                  },
```

Comment:
please note, that "minItems" does not change: we add 1 new file and allow 2 old files to move to a new folder